src/core/credrank/markovProcessGraph.test.js

```diff
@@ -356,16 +359,21 @@ describe("core/credrank/markovProcessGraph", () => {
 
   describe("to/froJSON", () => {
     it("has round trip equality", () => {
-      const mpg = markovProcessGraph();
-      const mpgJson = mpg.toJSON();
-      const mpg_ = MarkovProcessGraph.fromJSON(mpgJson);
-      const mpgJson_ = mpg_.toJSON();
-      expect(mpg).toEqual(mpg_);
-      expect(mpgJson).toEqual(mpgJson_);
+      const mpg1 = markovProcessGraph();
+      const mpgJson1 = mpg1.toJSON();
+      const mpg2 = markovProcessGraphParser.parseOrThrow(mpgJson1);
+      const mpgJson2 = mpg2.toJSON();
+      expect(mpg1).toEqual(mpg2);
+      expect(mpgJson1).toEqual(mpgJson2);
+    });
+    it("parser works", async () => {
+      const cg = await markovProcessGraph();
+      const cgJson = markovProcessGraph().toJSON();
+      expect(markovProcessGraphParser.parseOrThrow(cgJson)).toEqual(cg);
     });
     it("serialization does not change node/edge iteration order", () => {
       const mpg1 = markovProcessGraph();
-      const mpg2 = MarkovProcessGraph.fromJSON(mpg1.toJSON());
+      const mpg2 = markovProcessGraphParser.parseOrThrow(mpg1.toJSON());
```

Comment:
why did this change?

Reply:
Shouldn't have, initially I was playing around with removing `fromJSON` in favor of just using the parser, but I think keeping it is cleaner. Fixed.